### PR TITLE
Removed legacy transports and templates code

### PR DESCRIPTION
**Pull request by laf**
DO NOT MERGE UNTIL FEB

### CODE REVIEW

**Comment by murrant** [comment]
Does this actually remove legacy transports?  Or just templates?

File: misc/notifications.rss [hunk 1]
```diff
@@ -48,5 +48,10 @@
    enable this. Please note that the default is off to match with the other default contact settings. If you expect normal users to receive alerts then you should enable this new option.</description>
    <pubDate>Mon, 15 May 2017 23:00:00 +0000</pubDate>
  </item>
+  <item>
+    <title>Legacy Alert Templates and Transports removed</title>
+    <description>As described previously, Legacy Alert Templates and Transports have been removed.  You can find more info here: https://community.librenms.org/t/deprecation-notice-alerting-legacy-transports-and-templates/5915</description>
+    <pubDate>Feb, 13 May 2017 23:00:00 +0000</pubDate>
```
Comment (TheGreatDoc) [inline misc/notifications.rss]: Seems you forgot to change pubDate. It should be `Fri, 15 Feb 2017 23:00:00 +0000`

Comment (murrant) [reply misc/notifications.rss]: Sure, go ahead and update that.

**Comment by TheGreatDoc** [comment]
@murrant you left the year untouched 😄

**Comment by murrant** [comment]
Oopsie

**Pull request merged**